libs/coin-bitcoin/src/cli-transaction.ts

```diff
@@ -38,15 +39,22 @@ function inferTransactions(
   const feePerByte = new BigNumber(opts.feePerByte === undefined ? 1 : opts.feePerByte);
   return transactions.map(({ transaction }) => {
     invariant(transaction.family === "bitcoin", "bitcoin family");
+
+    const bitcoinStragegy = opts["bitcoin-pick-strategy"];
+    if (typeof bitcoinStragegy !== "string" && !(bitcoinStragegy in bitcoinPickingStrategy)) {
```

Comment:
I guess it should be "||" other than "&&"?
and maybe you can indicate the error message such as throw new Error("bitcoin picking strategy not found");